docs/extensions/index.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Extensions
+has_children: true
+nav_order: 7
+---
+
+# ONNXRuntime-Extensions
+
+[![Build Status](https://dev.azure.com/onnxruntime/onnxruntime/_apis/build/status%2Fmicrosoft.onnxruntime-extensions?branchName=main)](https://dev.azure.com/onnxruntime/onnxruntime/_build/latest?definitionId=209&branchName=main)
+
+## What is ONNXRuntime-Extensions?
+
+Introduction: ONNXRuntime-Extensions is a library that extends the capability of the ONNX models and inference with ONNX Runtime, via ONNX Runtime Custom Operator ABIs. It includes a set of [ONNX Runtime Custom Operators](https://onnxruntime.ai/docs/reference/operators/add-custom-op.html) to support the common pre- and post-processing operators for audio, vision, text, and NLP models. ONNXRuntime-Extensions also supports multiple languages and platforms, like Python on Windows/Linux/macOS, some mobile platforms like Android and iOS, and Web-Assembly etc. The basic workflow is to enhance a ONNX model firstly and then do the model inference with ONNX Runtime and ONNXRuntime-Extensions package.
+
+
+<img src="../../images/combine-ai-extensions-img.png" alt="Pre and post-processing custom operators for vision, text, and NLP models" width="100%"/>
+<sub>This image was created using <a href="https://github.com/sayanshaw24/combine" target="_blank">Combine.AI</a>, which is powered by Bing Chat, Bing Image Creator, and EdgeGPT.</sub>
+
+### Quickstart
+
+## Installation
+### **Python installation**
```

Comment:
Add other package formats: nuget, android ios etc

Reply:
added it for NuGet! for the mobile installations, i am not super sure myself and don't want to mess it up, @skottmckay @edgchen1 could one of you assist with this?